server/templates/details.html.tmpl

```diff
@@ -56,6 +56,24 @@
 <li data-status="{{$s}}">
   <div class="bg-white p-2 shadow-sm max-w-lg status-stripe {{$s}}">
     {{template "result-details" .}}
+    {{if (or .PredicateResults (hasRequires .Requires))}}
+    <details class="bg-light-gray5 p-2 mt-2 text-sm">
+      <summary class="text-dark-gray3 font-semibold">Details</summary>
+      <div class="pt-2"></div>
+      <div class="border-t border-light-gray3">
+       {{if .PredicateResults}}
+          <div class="pt-2 mt-2">
```

Comment:
I think this only needs `pt-2` now that the border is defined on a different element. That will also match the requires `div` below.